src/styles/variables.scss

```diff
@@ -0,0 +1,2 @@
+$main: #0065f2;
```

Comment:
Can you customise using the Bulma variables as shown here https://bulma.io/documentation/customize/variables/ and as done in all.scss